src/emqtt_sock.erl

```diff
@@ -65,9 +65,9 @@ ssl_upgrade(Host, Sock, SslOpts, Timeout) ->
     SslOpts4 = apply_host_check_fun(SslOpts3),
     case ssl:connect(Sock, SslOpts4, Timeout) of
         {ok, SslSock} ->
-            ok = ssl:controlling_process(SslSock, self()),
```

Comment:
no need to call this here.
`ssl_upgrade` is called immediately after `gen_tcp:connect` there is no controlling process switch.